js/modules/timeline.src.js

```diff
@@ -447,18 +478,25 @@ undocumentedSeriesType('timeline', 'line',
                 radius = 0,
                 attribs;
 
+            // Call default markerAttribs method, when the xAxis type
+            // is set to datetime.
+            if (series.xAxis.isDatetimeAxis) {
+                return H.seriesTypes.line.prototype.markerAttribs
+                    .call(this, point, state);
+            }
+
             // Handle hover and select states
             if (state) {
                 seriesStateOptions = seriesMarkerOptions.states[state] || {};
                 pointStateOptions = pointMarkerOptions.states &&
-                pointMarkerOptions.states[state] || {};
+                    pointMarkerOptions.states[state] || {};
 
                 radius = pick(
                     pointStateOptions.radius,
                     seriesStateOptions.radius,
                     radius + (
                         seriesStateOptions.radiusPlus ||
-                    0
+                        0
                     )
```

Comment:
The above 4 lines fits well under 80 characters line limit -> `radius + (seriesStateOptions.radiusPlus || 0)`